alien4cloud-wizard-ui/src/main/webapp/app/shared/pipes/button-label.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ApplicationOverview, DeploymentStatus } from '@app/core';
+
+@Pipe({
+  name: 'buttonLabel'
+})
+export class ButtonLabelPipe implements PipeTransform {
```

Comment:
- Rename the class to something more explicite, for example : DeploymentStatusButtonLabel 
- add a w4c prefix to the pipe name